include/rocksdb/db.h

```diff
@@ -1587,6 +1587,15 @@ class DB {
     return Status::NotSupported("EndTrace() is not implemented.");
   }
 
+  virtual Status StartIOTrace(Env* /*env*/, const TraceOptions& /*options*/,
```

Comment:
I think these methods also need to be added to the StackableDB

Reply:
Thank you for pointing that out. I will add to StackableDB as well.